azerothcore/azerothcore-wotlk
# fix(DB): Arcanist Helion missing gossip menu text

## Changes Proposed:
-  Added his gossip menu text

## Issues Addressed:

- Closes #6356 

## SOURCE:

https://www.youtube.com/watch?v=4tof9x0Zqr0&ab_channel=CauthonLuck

**Minute 5:57**
## Tests Performed:

- Tested in-game Windows 10

## How to Test the Changes:


1. Create a new Horde character
2. .go c 55055
3. Talk to the NPC and check his gossip text:
_"Knowledge is power - TRUE power, my young friend. You'll be wise to acquire as much of it as you can, and pay proper heed to those who have already done so.
Before the razing of the Sunwell, we fooled ourselves into thinking we had neared the apex of our civilization. It took the Scourge to bring us to our knees... and in a way, back to reality."_

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
